sorc/global_cycle.fd/cycle.f90

```diff
@@ -688,25 +724,65 @@ SUBROUTINE SFCDRV(LUGB, IDIM,JDIM,LENSFC,LSOIL,DELTSFC,  &
         SLC_BCK = SLCFCS
 
         ! below updates [STC/SMC/STC]FCS to hold the analysis
-        CALL ADD_INCREMENT_SOIL(RLA,RLO,STCFCS,SMCFCS,SLCFCS,STC_UPDATED, SLC_UPDATED, &
+        ! ONLY isolate soil temperature and soil moisture ingest/adjust routines in
```

Comment:
We need to have the same process for both increment types (since the intention of the JEDI increments is to test what we should do for the GSI), so either need to code the new method for the GSI, or make the JEDI method the same as the GSI. 

If "c) ingest soil moisture incr. based on newly-adjusted soil state due to b)" implies that JEDI is being called twice (as in the second call uses the updated soil moisture from b, then we definitely don't want to do it this way (changes to the soil moisture from a and b will be small, and certainly doesn't justify the cost/difficulty of calling JEDI twice).

Reply:
Just saw below that JEDI is not re-called, which is good.